src/service.ui/index.js

```diff
@@ -216,15 +191,88 @@ export default class UIService {
   setColor(classes, color) {
     return classes?.replaceAll("{color}", color);
   }
+
+  /**
+   Set theme css variables.
+   @param { Object } config
+   @returns { void }
+   */
+  setTheme = (config = {}) => {
+    const darkMode = config?.darkMode || vuelessConfig?.darkMode || DEFAULT_DARK_MODE;
```

Comment:
We probably should check if `darkMode !== undefined` here. If darkMode is undefined, we will look in `localStoarage` and `window.matchMedia("(prefers-color-scheme: dark)");` to set theme  preferred by user or browser.